src/statespacemodel.jl

```diff
@@ -0,0 +1,141 @@
+immutable StateSpaceModel{T <: Real}
+
+    # Process transition matrix, control matrix, and noise covariance
+    F::Function
+    B::Matrix{T}
```

Comment:
Sure - I need to put in a bit more thought about how EM estimation of time-independent parameters will best integrate with time-dependent system matrices, but the Kalman filtering / smoothing part should be simple.